index.js

```diff
@@ -112,6 +112,16 @@
     };
   }
 
+  //  clean :: Object -> Object
+  function clean(x) {
+    var o = Object.create(null);
+    for (var k in x) {
+      /* istanbul ignore else */
+      if (has(k, x)) o[k] = x[k];
+    }
+    return o;
+  }
+
```

Comment:
Ahh, that's how they are sorted. Of course! I tried to identify some kind of categorization and appended the function to what intuitively felt like a category. ;)
:zap: